wasm/src/lib.rs

```diff
@@ -37,6 +38,34 @@ pub fn git_hash() -> String {
     git_hash.into()
 }
 
+fn get_qir_internal(code: &str) -> std::result::Result<String, (Error, Vec<Frame>)> {
+    let core = compile::core();
+    let mut store = PackageStore::new(core);
+    let std = compile::std(&store, TargetProfile::Base);
+    let std = store.insert(std);
+    let sources = SourceMap::new([("test".into(), code.into())], None);
+
+    let unit = qsc::compile::compile(
+        &store,
+        &[std],
+        sources,
+        PackageType::Exe,
+        TargetProfile::Base,
+    );
+    let package = store.insert(unit.0);
```

Comment:
This should check for errors before inserting into the store. Putting a CompileUnit into the store when it has errors is unsafe. I vaguely recall there being a panic in the PackageStore insert method that would have prevented this but it might have been removed for some reason (perhaps for the language server?).